### PR TITLE
docs: add more info how to build and debug in Getting Started

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: docs/user/getting_started.rst [hunk 1]
```diff
@@ -86,11 +86,19 @@ Next go back to the top-level Gluon directory and build Gluon::
...
 
-In case of errors read the messages carefully and try to fix the stated issues (e.g. install tools not available yet).
+In case of errors read the messages carefully and try to fix the stated issues
+(e.g. install missing tools not available or look at the wiki:
+[Troubleshooting](https://github.com/freifunk-gluon/gluon/wiki/Troubleshooting))
 
 ``ar71xx-generic`` is the most common target and will generate images for most of the supported hardware.
 To see a complete list of supported targets, call ``make`` without setting ``GLUON_TARGET``.
 
+To build all (currently 19) targets use a loop like this:
```
Comment (mweinelt) [inline docs/user/getting_started.rst]: Nope, we're not going to maintain that list in the documentation. Please can write their own scripts using the output of `make list-targets` though.

Comment (rubo77) [reply docs/user/getting_started.rst]: good point! fixed

File: docs/user/getting_started.rst [hunk 2]
```diff
@@ -86,11 +86,19 @@ Next go back to the top-level Gluon directory and build Gluon::
...
+In case of errors read the messages carefully and try to fix the stated issues
+(e.g. install missing tools not available or look at the wiki:
+[Troubleshooting](https://github.com/freifunk-gluon/gluon/wiki/Troubleshooting))
 
 ``ar71xx-generic`` is the most common target and will generate images for most of the supported hardware.
 To see a complete list of supported targets, call ``make`` without setting ``GLUON_TARGET``.
 
+To build all targets use a loop like this:
+
+    for i in $(make BROKEN=1 list-targets); do echo $i; done
```
Comment (mweinelt) [inline docs/user/getting_started.rst]: > do echo $i; done

probably remains from your tests

Comment (mweinelt) [reply docs/user/getting_started.rst]: We will never recommend building with BROKEN=1.

Comment (rubo77) [reply docs/user/getting_started.rst]: fixed

File: docs/user/getting_started.rst [hunk 3]
```diff
@@ -86,11 +86,19 @@ Next go back to the top-level Gluon directory and build Gluon::
...
+In case of errors read the messages carefully and try to fix the stated issues
+(e.g. install missing tools not available or look at the wiki:
+[Troubleshooting](https://github.com/freifunk-gluon/gluon/wiki/Troubleshooting))
 
 ``ar71xx-generic`` is the most common target and will generate images for most of the supported hardware.
 To see a complete list of supported targets, call ``make`` without setting ``GLUON_TARGET``.
 
+To build all targets use a loop like this:
+
+    for TARGET in $(make list-targets); do echo $i; done
```
Comment (mweinelt) [inline docs/user/getting_started.rst]: Still `echo $i; done` at the end of this line.

Comment (rubo77) [reply docs/user/getting_started.rst]: done

**Pull request merged**